### DISCUSSION TITLE
Does "字" really refers to "names" in this sentence?

### DISCUSSION THREAD

**Discussion by rodrigo-morales-1** [rodrigo-morales-1/subtitles-npcr#1135] [Q&A] [answered]
In the sentence shown below, the current translation of 字 is "names", but I'm not sure if this should be the current translation. I know the word 名字 refers to "name" and includes the character 字, but in that sentence, the character 字 could also refer to a Chinese character that means making a promise. For example, 应 from the word 答应.

https://github.com/rdrg109/subtitles-npcr/blob/ec84d0c7642202f2c2314e16ed48a989f8e4e5ab/sentences/46-text-2.yaml#L304-L308

**Comment by rodrigo-morales-1** [comment]
@fhxshrs in #1121 you reviewed this sentence, so you might know the answer to this question. Could you confirm please?

**Reply by rodrigo-morales-1** [reply, marked as answer]
I just learned that 签字 means "to sign one's name", so that's the reason why 字 refers to "names" in this sentence.

@fhxshrs please omit this ping.

**Reply by fhxshrs** [reply]
Great!
